### PR TITLE
rename to grafana-helm

### PR DESCRIPTION
**Pull request by splattner**
Rename component to grafana-helm
simplify helm values handling

### CODE REVIEW

File: class/defaults.yml [hunk 1]
```diff
@@ -1,23 +1,16 @@
 parameters:
-  grafana:
+  grafana-helm:
```
Comment (srueg) [inline class/defaults.yml:2]:
```suggestion
  grafana_helm:
```

File: class/grafana-helm.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+parameters:
+  kapitan:
+    dependencies:
+      - type: helm
+        source: https://grafana.github.io/helm-charts
+        chart_name: grafana
+        version: ${grafana-helm:charts:grafana}
```
Comment (srueg) [inline class/grafana-helm.yml:7]:
```suggestion
        version: ${grafana_helm:charts:grafana}
```

File: class/grafana-helm.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        input_type: jsonnet
+        output_path: apps/
+      - input_paths:
+          - grafana-helm/component/main.jsonnet
+        input_type: jsonnet
+        output_path: grafana/
+      - input_type: helm
+        input_paths:
+          - grafana-helm/helmcharts/grafana
+        helm_values: ${grafana-helm:helm_values}
```
Comment (srueg) [inline class/grafana-helm.yml:21]:
```suggestion
        helm_values: ${grafana_helm:helm_values}
```

File: class/grafana-helm.yml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+        helm_params:
+          namespace: ${grafana-helm:namespace}
```
Comment (srueg) [inline class/grafana-helm.yml:23]:
```suggestion
          namespace: ${grafana_helm:namespace}
```

File: component/app.jsonnet [hunk 5]
```diff
@@ -1,10 +1,10 @@
 local kap = import 'lib/kapitan.libjsonnet';
 local inv = kap.inventory();
-local params = inv.parameters.grafana;
+local params = inv.parameters.grafana-helm;
```
Comment (srueg) [inline component/app.jsonnet:3]:
```suggestion
local params = inv.parameters.grafana_helm;
```

File: component/main.jsonnet [hunk 6]
```diff
@@ -3,7 +3,7 @@ local kap = import 'lib/kapitan.libjsonnet';
 local kube = import 'lib/kube.libjsonnet';
 local inv = kap.inventory();
 // The hiera parameters for the component
-local params = inv.parameters.grafana;
+local params = inv.parameters.grafana-helm;
```
Comment (srueg) [inline component/main.jsonnet:6]:
```suggestion
local params = inv.parameters.grafana_helm;
```

File: docs/modules/ROOT/pages/references/parameters.adoc [hunk 7]
```diff
@@ -1,6 +1,6 @@
 = Parameters
 
-The parent key for all of the following parameters is `grafana`.
+The parent key for all of the following parameters is `grafana-helm`.
```
Comment (srueg) [inline docs/modules/ROOT/pages/references/parameters.adoc:3]:
```suggestion
The parent key for all of the following parameters is `grafana_helm`.
```

**Review by srueg — CHANGES_REQUESTED** [review]

File: class/defaults.yml [hunk 8]
```diff
@@ -1,23 +1,16 @@
...
-      - chart-example.local
-      tls: []
-      annotations: {}
-
-
+    helm_values:
+      images:
+        grafana:
+          image: grafana/grafana
+          tag: ' 7.3.5'
```
Comment (srueg) [inline class/defaults.yml:10]:
```suggestion
          tag: '7.3.5'
```

Comment (srueg) [reply class/defaults.yml:10]: I think we can just remove the settings regarding the image here and use the defaults?

File: class/grafana-helm.yml [hunk 9]
```diff
@@ -0,0 +1,24 @@
+parameters:
+  kapitan:
+    dependencies:
+      - type: helm
+        source: https://grafana.github.io/helm-charts
+        chart_name: grafana
+        version: ${grafana_helm:charts:grafana}
+        output_path: dependencies/grafana/helmcharts/grafana
```
Comment (srueg) [inline class/grafana-helm.yml:8]:
```suggestion
        output_path: dependencies/grafana-helm/helmcharts/grafana
```

File: class/grafana-helm.yml [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+    compile:
+      - input_paths:
+          - grafana-helm/component/app.jsonnet
+        input_type: jsonnet
+        output_path: apps/
+      - input_paths:
+          - grafana-helm/component/main.jsonnet
+        input_type: jsonnet
+        output_path: grafana/
```
Comment (srueg) [inline class/grafana-helm.yml:17]:
```suggestion
        output_path: grafana-helm/
```

File: class/grafana-helm.yml [hunk 11]
```diff
@@ -0,0 +1,24 @@
...
+      - input_type: helm
+        input_paths:
+          - grafana-helm/helmcharts/grafana
+        helm_values: ${grafana_helm:helm_values}
+        helm_params:
+          namespace: ${grafana_helm:namespace}
```
Comment (srueg) [inline class/grafana-helm.yml:23]:
```suggestion
         release_name: ${_instance}
          namespace: ${grafana_helm:namespace}
```

File: component/app.jsonnet [hunk 12]
```diff
@@ -1,10 +1,10 @@
 local kap = import 'lib/kapitan.libjsonnet';
 local inv = kap.inventory();
-local params = inv.parameters.grafana;
+local params = inv.parameters.grafana_helm;
 local argocd = import 'lib/argocd.libjsonnet';
 
-local app = argocd.App('grafana', params.namespace);
+local app = argocd.App('grafana_helm', params.namespace);
```
Comment (srueg) [inline component/app.jsonnet:6]:
```suggestion
local app = argocd.App('grafana-helm', params.namespace);
```

File: component/app.jsonnet [hunk 13]
```diff
@@ -1,10 +1,10 @@
...
 
 {
-  grafana: app,
+  grafana_helm: app,
```
Comment (srueg) [inline component/app.jsonnet:9]:
```suggestion
  'grafana-helm': app,
```

**Review by srueg — CHANGES_REQUESTED** [review]

File: component/app.jsonnet (hunk 13 above)
Comment (splattner) [reply component/app.jsonnet]: I get an error with this `STATIC ERROR: component/app.jsonnet:9:3-10: expected one of :, ::, :::, +:, +::, +:::, got: -` in `lint_jsonet`

**Review by srueg — APPROVED** [review]

**Comment by srueg** [comment]
Relates to https://github.com/projectsyn/commodore/issues/274

**Pull request merged**